temp.py

```diff
@@ -0,0 +1,45 @@
+from bs4 import BeautifulSoup
+import requests
+import pandas as pd
+username=input('Enter Username')
+page=requests.get("https://www.codechef.com/users/"+username)
```

Comment:
Add a browser user agent, or the scraper will be detected as a bot and blocked.
```suggestion
page=requests.get("https://www.codechef.com/users/"+username, headers={'User-Agent': 'Mozilla/5.0'})
```